masterlist.yaml

```
@@ -6283,22 +6283,20 @@ plugins:
       - crc: 0xC32BA914
         util: 'SSEEdit v4.0.3'
 
-  - name: 'TouringCarriages.esp'
+  - name: 'TouringCarriages(-ICAIO)?.esp'
```

Same as the other PR, it should cover any future patches/addons.

```suggestion
  - name: 'TouringCarriages.*\.esp'
```